src/api/listsApi.ts

```diff
@@ -51,6 +52,14 @@ export class ListsApi extends BaseApi {
         );
     }
 
+    public async get(listId: number) {
```

Comment:
```suggestion
    public async get(request: ById) {
```
Чтобы единообразие интерфейсов сохранить.